via/static/scripts/video_player/components/VideoPlayer.tsx

```diff
@@ -0,0 +1,35 @@
+export type VideoPlayerProps = {
+  videoId: string;
+
+  /** Set the current timestamp of the video. */
+  time: number;
```

Comment:
This timestamp and the callbacks below are not actually used yet. They will be once more of the UI is fleshed out in subsequent PRs.